subgraphs/notional-finance/schema.graphql

```diff
@@ -0,0 +1,1290 @@
+
+  " Version of the subgraph implementation, in SemVer format (e.g. 1.0.0) "
+  subgraphVersion: String!
+
+  " Version of the methodology used to compute metrics, loosely based on SemVer format (e.g. 1.0.0) "
+  methodologyVersion: String!
+
+  " The blockchain network this subgraph is indexing on "
+  network: Network!
+
+  " The type of protocol (e.g. DEX, Lending, Yield, etc) "
+  type: ProtocolType!
+
+  " The specific lending protocol type "
+  lendingType: LendingType
+
+  " Risk type of the lending protocol "
+  riskType: RiskType
+
+  " Tokens that can be minted. Only applies to CDP "
+  mintedTokens: [Token!]
+
+  ##### Quantitative Data #####
+
+  " Number of cumulative unique users "
+  cumulativeUniqueUsers: Int!
+
+  " Number of cumulative depositors "
+  cumulativeUniqueDepositors: Int!
+
+  " Number of cumulative borrowers "
+  cumulativeUniqueBorrowers: Int!
+
+  " Number of cumulative liquidators (accounts that performed liquidation) "
+  cumulativeUniqueLiquidators: Int!
+
+  " Number of cumulative liquidatees (accounts that got liquidated) "
+  cumulativeUniqueLiquidatees: Int!
+
+  " Current TVL (Total Value Locked) of the entire protocol "
+  totalValueLockedUSD: BigDecimal!
+
+  " Current PCV (Protocol Controlled Value). Only relevant for protocols with PCV. "
+  protocolControlledValueUSD: BigDecimal
+
+  " Revenue claimed by suppliers to the protocol. LPs on DEXs (e.g. 0.25% of the swap fee in Sushiswap). Depositors on Lending Protocols. NFT sellers on OpenSea. "
+  cumulativeSupplySideRevenueUSD: BigDecimal!
+
+  " Gross revenue for the protocol (revenue claimed by protocol). Examples: AMM protocol fee (Sushi’s 0.05%). OpenSea 10% sell fee. "
+  cumulativeProtocolSideRevenueUSD: BigDecimal!
+
+  " All revenue generated by the protocol. e.g. 0.30% of swap fee in Sushiswap, all yield generated by Yearn. "
+  cumulativeTotalRevenueUSD: BigDecimal!
+
+  " Current balance of all deposited assets, in USD. Note this metric should be the same as TVL. "
+  totalDepositBalanceUSD: BigDecimal!
+
+  " Sum of all historical deposits in USD (only considers deposits and not withdrawals) "
+  cumulativeDepositUSD: BigDecimal!
+
+  " Current balance of all borrowed/minted assets (not historical cumulative), in USD. "
+  totalBorrowBalanceUSD: BigDecimal!
+
+  " Sum of all historical borrows/mints in USD (i.e. total loan origination). "
+  cumulativeBorrowUSD: BigDecimal!
+
+  " Sum of all historical liquidations in USD "
+  cumulativeLiquidateUSD: BigDecimal!
+
+  " Total supply of minted tokens in native amounts, with same ordering as mintedTokens. Only applies to CDP "
+  mintedTokenSupplies: [BigInt!]
+
+  " Total number of pools "
+  totalPoolCount: Int!
+
+  " Total number of open positions "
+  openPositionCount: Int!
+
+  " Total number of positions (open and closed) "
+  cumulativePositionCount: Int!
+
+  ##### Account / Position Data #####
+
+  " Any account that has made a deposit before "
+  depositors: [Account!]!
```

Comment:
Make these prefixed with `_` and change the Account / Position Data header to "Add Ons"

Reply:
Updated but weren't these part of the schema previously? Don't remember but I think I didn't add these for my purpose.

Reply:
Ah, nvm, I introduced these for usage metrics calculations.